boojum-cuda/native/barycentric.cu

```diff
@@ -162,4 +162,4 @@ EXTERN __launch_bounds__(1024, 1) __global__ void batch_barycentric_partial_redu
   batch_barycentric_partial_reduce_impl<extension_field>(batch_ys, lagrange_coeffs, partial_sums, log_count, num_polys);
 }
 
-} // namespace goldilocks
\ No newline at end of file
+} // namespace goldilocks
```

Comment:
This looks like some unrelated formatting change.

Reply:
fixed